documentation/src/docs/catalog.md

```diff
@@ -30,7 +63,7 @@ Pravega uses terms such as *streams* and *scopes* for managing streaming data, b
 
 When using Schema Registry serialization, further information is required in order to describe how to map binary data onto table columns. We have introduced an internal format factory `PravegaRegistryFormatFactory`. Currently it supports Json and Avro formats without any additional encryption and compression codecs.
 
-With the help of schema registry service, it is feasible to map Pravega streams to Flink tables as the following table shows:
+With the help of schema Registry service, it is feasible to map Pravega streams to Flink tables as the following table shows:
 
 | Flink Catalog terms                  | Pravega terms     |
 |--------------------------------------|-------------------|
```

Comment:
I think in the below table, the word `name` is redundant.

Reply:
fixed